src/types/types.ts

```diff
@@ -670,6 +670,7 @@ export interface EdgeSpendInfo {
   /** Enables/disables RBF for currencies where RBF is opt-in or optional. */
   enableRbf?: boolean
   pendingTxs?: EdgeTransaction[]
+  /** @deprecated Use enableRbf instead */
```

Comment:
The `enableRbf` flag isn't right alternative. This comment should say "Use `EdgeCurrencyWallet.accelerate` instead"